binding.gyp

```diff
@@ -1,63 +1,70 @@
 {
-  'targets': [
+  "targets": [
```

Comment:
All of the changes in this file were to get Windows to publish.  It’s mostly changes from `”` to `’`.  

The only other meaningful changes are noted below.

Reply:
Updated.  I reverted all the quote changes.  The windows build error was coming from somewhere else.  I noticed an indentation inconsistency in the file so when I was in there editing I reverted the quotes to the more standard `'`.